apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -43,16 +44,36 @@ export default class ReviewTab extends Component {
       });
   };
 
+  onCommentDelete = deletedCommentId => {
+    const comments = [...this.state.comments];
+    _.remove(comments, comment => comment.id === deletedCommentId);
+
+    this.setState({comments: comments});
+  };
+
+  onCommentResolve = resolvedCommentId => {
```

Comment:
this name is slightly confusing, because it implies it always resolves a comment when really it toggles the resolved state. We have `onResolve` a few places, should it instead be something along the lines of `onResolveStateToggle`?